test/fixture-bootstrap/refer-to-other-source/terragrunt_files/.tgf.config

```diff
@@ -0,0 +1 @@
+docker-image-version: 2.8
```

Comment:
```suggestion
docker-image-version: 2.8

```

Reply:
Oops, I actually forgot the .tgf.config. It shouldn't be there